smoke_test_scripts/assets/example_config.yaml

```diff
@@ -0,0 +1,7 @@
+all-commands: false
+config: ""
+debug: false
+endpoint: ""
```

Comment:
This value (`endpoint`) ends up being set (in actual config file named `drconfig.yaml` in respective directory) during smoke-test flow and we validate that it's what we expect.